Assets/MRTK/Core/Utilities/Gltf/Serialization/GltfConversions.cs

```diff
@@ -218,7 +218,7 @@ public static int[] GetIntArray(this GltfAccessor accessor, bool flipFaces = tru
 
             var array = new int[accessor.count];
 
-            GetTypeDetails(accessor.componentType, out int componentSize, out float _);
+            GetTypeDetails(accessor.ComponentType, out int componentSize, out float _);
```

Comment:
Can you think of any way to make this a non-breaking change? I'm not coming up with anything at the moment, since the (newly private) backing fields have to match the file's field name (unless there's a tag or something to say "this field is serialized with this name even though it's named differently"?)

Reply:
I tried but noting came to mind (I even remembering trying FormerlySerializedAs with no luck)